src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsKernel.hpp

```diff
@@ -344,9 +341,17 @@ class Multiphase :
 
     for( integer a = 0; a < numNodesPerElem; ++a )
     {
-      stack.localDispFlowJacobian[a*3+0][0] += dNdX[a][0] * dTotalStress_dPressure * detJxW;
-      stack.localDispFlowJacobian[a*3+1][0] += dNdX[a][1] * dTotalStress_dPressure * detJxW;
-      stack.localDispFlowJacobian[a*3+2][0] += dNdX[a][2] * dTotalStress_dPressure * detJxW;
+      stack.localDispFlowJacobian[a*3+0][0] += dNdX[a][0] * dTotalStress_dPressure[0] * detJxW;
+      stack.localDispFlowJacobian[a*3+0][0] += dNdX[a][2] * dTotalStress_dPressure[4] * detJxW;
+      stack.localDispFlowJacobian[a*3+0][0] += dNdX[a][1] * dTotalStress_dPressure[5] * detJxW;
+
+      stack.localDispFlowJacobian[a*3+1][0] += dNdX[a][1] * dTotalStress_dPressure[1] * detJxW;
+      stack.localDispFlowJacobian[a*3+1][0] += dNdX[a][2] * dTotalStress_dPressure[3] * detJxW;
+      stack.localDispFlowJacobian[a*3+1][0] += dNdX[a][0] * dTotalStress_dPressure[5] * detJxW;
+
+      stack.localDispFlowJacobian[a*3+2][0] += dNdX[a][2] * dTotalStress_dPressure[2] * detJxW;
+      stack.localDispFlowJacobian[a*3+2][0] += dNdX[a][1] * dTotalStress_dPressure[3] * detJxW;
+      stack.localDispFlowJacobian[a*3+2][0] += dNdX[a][0] * dTotalStress_dPressure[4] * detJxW;
```

Comment:
`dTotalStress_dPressure[i]` with `i=3,4,5` are always nil. So we can simplify to:
```suggestion
      stack.localDispFlowJacobian[a*3+0][0] += dNdX[a][0] * dTotalStress_dPressure[0] * detJxW;
      stack.localDispFlowJacobian[a*3+1][0] += dNdX[a][1] * dTotalStress_dPressure[1] * detJxW;
      stack.localDispFlowJacobian[a*3+2][0] += dNdX[a][2] * dTotalStress_dPressure[2] * detJxW;
```